cypress/integration/settings/email-change.js

```diff
@@ -0,0 +1,43 @@
+/* global cy */
+describe('Email input field', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/settings');
+  });
+
+  it('Should be possible to type new email', () => {
+    cy.get('[id=new-email]')
+      .type('bar@foo.com')
+      .should('have.attr', 'value', 'bar@foo.com');
+  });
+
+  it('Should be possible to confirm email', () => {
+    cy.get('[id=confirm-email]')
+      .type('bar@foo.com')
+      .should('have.attr', 'value', 'bar@foo.com');
+  });
+
+  it('Should be possible to save new email', () => {
+    cy.contains('Save').click({ force: true });
```

Comment:
I could be wrong here, because it's been a while since I messed with Cypress, but wouldn't using `force: true` essentially mean this test doesn't ever fail?

Reply:
You are correct I will change this.